optaplanner-examples/src/test/java/org/optaplanner/examples/cloudbalancing/app/CloudBalancingDaemonTest.java

```diff
@@ -51,9 +50,8 @@
     private volatile Throwable solverThreadException = null;
     private volatile CloudBalance currentBestSolution = null;
 
-    @Disabled("PLANNER-2249")
     @Test
-    @Timeout(600)
+    @Timeout(1500)
```

Comment:
15 above, 1500 here? That is quite the range!
Also, 1500 seconds, seriously? This test needs to become a turtle test and not be run within the default test suite.

Reply:
Most timeouts for our optpalanner-examples tests (not the turtle tests) are 600 seconds, yet they run for 5 seconds on most machines. That's intentional, because we sometimes saw them randomly break if was only 60 seconds because jenkins slaves had unreliable cpu.